test/unit/test-history.js

```diff
@@ -919,6 +919,11 @@ describes.sandboxed('HistoryBindingVirtual', {}, () => {
       expect(onStateUpdated).to.be.calledOnce;
       expect(onStateUpdated).to.be.calledWithMatch({stackIndex: 123});
     });
+
+    it('sends invalid data', () => {
```

Comment:
What?

Reply:
If the viewer sends invalid data in "historyPopped" message. See other unit tests above.